licenses/GPL-2.0-or-later.txt

```diff
@@ -0,0 +1,15 @@
+    Copyright (C) <year>  <name of author>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+
+The license text can be found in GPL-2.0.txt.
```

Comment:
`GPL-2.0.txt` seems to be missing from this PR.

Reply:
Will add - thanks @shermp (keeping unresolved until added).